cmd/distro/write_strings.go

```diff
@@ -10,15 +10,14 @@ import (
 	"github.com/pingcap/log"
 	"go.uber.org/zap"
 
-	_ "github.com/pingcap/tidb-dashboard/internal/resource/distrores"
 	"github.com/pingcap/tidb-dashboard/util/distro"
 )
 
 func main() {
 	outputPath := flag.String("o", "", "Output json file path")
 	flag.Parse()
 
-	d, _ := json.Marshal(distro.R())
+	d, _ := json.Marshal(distro.DefaultDistroRes)
```

Comment:
Move this file to cmd/distro/write_strings.go, so the `scripts` doesn't need to depend on tidb-dashboard, not sure whether it is a good idea.

Reply:
I think it is not a good idea. It is not a runnable binary that we ever want to provide to the user, so `cmd` is not a right place for it.

Actually it's fine to make `scripts/go.mod` depending on the tidb-dashboard, as bumping the `scripts/go.sum` is as easy as running a `go mod tidy`, which can be automated and encapsulated in something like scripts.